apps/gauzy/src/app/pages/dashboard/dashboard-switcher/dashboard-switcher.component.scss

```
@@ -25,6 +25,18 @@
 			display: flex;
 			align-items: center;
 			gap: 8px;
+			// `overflow-x: auto` promotes this row's `overflow-y` from `visible`
```

Stylelint (17.14.1)

[error] Unknown word `overflow-x (CssSyntaxError)

(CssSyntaxError)